scripts/start.sh

```diff
@@ -1,25 +1,30 @@
 #!/bin/bash
-echo "---Checking if UID: ${UID} matches user---"
+echo "---Ensuring UID: ${UID} matches user---"
 usermod -u ${UID} ${USER}
-echo "---Checking if GID: ${GID} matches user---"
+echo "---Ensuring GID: ${GID} matches user---"
+groupmod -g ${GID} ${USER} 
 usermod -g ${GID} ${USER}
 echo "---Setting umask to ${UMASK}---"
 umask ${UMASK}
 
 echo "---Checking for optional scripts---"
-if [ -f /opt/scripts/user.sh ]; then
-	echo "---Found optional script, executing---"
-    chmod +x /opt/scripts/user.sh
-    /opt/scripts/user.sh
+cp -f /opt/custom/user.sh /opt/scripts/start-user.sh >> /dev/null ||:
+cp -f /opt/scripts/user.sh /opt/scripts/start-user.sh >> /dev/null ||:
+
+if [ -f /opt/scripts/start-user.sh ]; then
+    echo "---Found optional script, executing---"
+    chmod -f +x /opt/scripts/user.sh ||:
```

Comment:
Should be: `chmod -f +x /opt/scripts/start-user.sh ||:`

Reply:
Indeed.

Reply:
```suggestion
    chmod -f +x /opt/scripts/start-user.sh.sh ||:
```